samples/net/cloud/aws_iot_mqtt/src/main.c

```diff
@@ -54,6 +54,10 @@ static uint32_t messages_received_counter;
 static bool do_publish;	  /* Trigger client to publish */
 static bool do_subscribe; /* Trigger client to subscribe */
 
+#if (CONFIG_AWS_MQTT_PORT == 443 && !CONFIG_MQTT_LIB_WEBSOCKET)
```

Comment:
I'm not sure if `!CONFIG_MQTT_LIB_WEBSOCKET` would be a correct syntax, the symbol might be undefined. I think it would be better to use:
`#if (CONFIG_AWS_MQTT_PORT == 443) && !defined(CONFIG_MQTT_LIB_WEBSOCKET)`